test/proxy.deltas.test.ts

```diff
@@ -0,0 +1,372 @@
+import {ProxyDeltasTestFactory} from "../contract-bindings/ethers";
```

Comment:
Is there a reason why this file is named `proxy.deltas.test.ts` instead of `proxy-deltas.test.ts`?

Reply:
None at all, fixed